.travis.yml

```diff
@@ -16,4 +16,10 @@ jobs:
       name: "Unit Tests"
       script: npm run test:lint && npm run test:unit && npm run test:danger && npm run test:git-history
     - name: "Integration Tests"
-      script: ./scripts/run-integration-tests.travis.sh
\ No newline at end of file
+      script: ./scripts/run-integration-tests.travis.sh
+    - stage: release
+      if: branch = master
+      script:
```

Comment:
Do we have a task to publish the One App static assets to a CDN? Could potentially `npm publish` the statics only in order to leverage https://www.jsdelivr.com